src/App.js

```diff
@@ -15,25 +15,24 @@ import {PlayListPage} from "./pages/PlayList/PlayListPage";
 import {Page404} from "./pages/Page-404/Page404";
 import { SignUpPage } from "./pages/SignUpPage/SignUpPage";
 import { LogInPage } from "./pages/Log-In-Page/LogInPage";
-import axios from "axios";
+import {Toast} from "./components/Toast/Toast";
+import { useAuthorization } from "./contextAndReducers/AuthProvider";
+import {PrivateRoute} from "./components/PrivateRoute/PrivateRoute";
 function App() {
 const {modalDisplay, setModalDisplay}=useDataValues();
-  useEffect(
-       async () => {
-      try {
-        const response = await axios.post("/api/auth/login", {
-              email: "adarshbalika@gmail.com",
-    password: "adarshBalika123",
-        });
-        console.log(response.data.foundUser);
-        localStorage.setItem("token", response.data.encodedToken);
-      } catch (error) {
-        console.log(error);
-      }
-  
-  },[])
+const {authState,authDispatch }=useAuthorization();
+useEffect(()=>{
+ 
+const timeoutID= setTimeout(()=> {authDispatch({
+        type: "TOAST",
+        payload: { display:"none", message: "", type: "" },
+   
+      });clearTimeout(timeoutID)},2000)
```

Comment:
suggestion - this can be written a little bit cleaner

Reply:
Thanks for reviewing I will look into it